spec/system/field_of_operation_spec.rb

```diff
@@ -0,0 +1,59 @@
+RSpec.describe "Field of operation page" do
+  describe "GET /<document_type>/<slug>" do
+    let(:content_store_response) { GovukSchemas::Example.find("field_of_operation", example_name: "field_of_operation") }
+    let(:base_path) { content_store_response.fetch("base_path") }
+
+    before do
+      stub_content_store_has_item(base_path, content_store_response)
+      visit base_path
+    end
+
+    it "displays the page" do
+      expect(page.status_code).to eq(200)
+    end
+
+    it "has the correct title" do
+      expect(page.title).to eq("Iraq - GOV.UK")
+    end
+
+    it "has the correct heading, context, and locale" do
+      within("h1") do
+        expect(page).to have_text("Operations in Iraq")
+      end
+      within(".gem-c-heading__context") do
+        expect(page).to have_text("British fatalities")
+      end
+
+      locale = page.find("main div:first-of-type > .gem-c-heading")["lang"]
+      expect(locale).to eq content_store_response["locale"]
+    end
+
+    it "has the correct subheadings and text" do
```

Comment:
I think we can add some additional tests where the fatality notice heading and list aren't shown if there are no notices (when [`@content_item.fatality_notices.any?`](https://github.com/alphagov/government-frontend/blob/8d2872c29486c4806cdd134cf1eb0a931f2a146c/app/views/content_items/field_of_operation.html.erb#L34) returns `false`)

You could wrap the existing tests in a `context "when there are fatality notices present" do` and have a separate context for "when there are no fatality notices present".